library/src/test/kotlin/io/github/wax911/library/annotation/processor/fragment/FragmentRegexUtilTest.kt

```diff
@@ -0,0 +1,101 @@
+package io.github.wax911.library.annotation.processor.fragment
+
+import org.junit.Assert.assertEquals
```

Comment:
This isn't our project, and they don't use the Junit 5 or the other assertion libraries we use. I'm just sticking with what they have.